paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/source/ContinuousFileSplitEnumerator.java

```diff
@@ -71,13 +73,16 @@ public class ContinuousFileSplitEnumerator
 
     private boolean finished = false;
 
+    private final boolean unawareBucket;
+
     public ContinuousFileSplitEnumerator(
             SplitEnumeratorContext<FileStoreSourceSplit> context,
             Collection<FileStoreSourceSplit> remainSplits,
             @Nullable Long nextSnapshotId,
             long discoveryInterval,
             int splitBatchSize,
```

Comment:
Maybe we can just remove this splitBatchSize for simplify the `createAssignment`.